### PR TITLE
Feature checkout flow

### PR DESCRIPTION
**Pull request by vikrantrana1337**

### CODE REVIEW

File: src/components/Inputs/index.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      centered,
+      isValid,
+      hasLabel,
+      validation,
+      className,
+      containerClassName,
+      textClassName,
+      text,
+      dropdown,
+      textStyle,
```
Comment (GrgSaloni) [inline src/components/Inputs/index.js]: I think we wont be needing all these .. isiliye we have to remove all unused at end

File: src/components/header/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+// ASSETS
+import backArrow from "../../assets/back-common.svg";
+
+class Header extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bannervehicle: true,
+      subtext: true
+    };
```
Comment (GrgSaloni) [inline src/components/header/index.js]: Please set them false n then use wherever required. Moreover they are not getting used anywhere.

File: src/routes.js [hunk 3]
```diff
@@ -16,6 +17,7 @@ const AppRouter = (props) => (
       <div>
         <Switch>
           <Route exact path="/" component={Home} />
+          <Route path="/edit-form" component={AddressForm} />
```
Comment (GrgSaloni) [inline src/routes.js]: Modify route name as suggested over the mesage

File: src/components/Inputs/index.js (hunk 1 above)
Comment (vikrantrana1337) [reply src/components/Inputs/index.js]: done

File: src/components/header/index.js (hunk 2 above)
Comment (vikrantrana1337) [reply src/components/header/index.js]: done

**Pull request merged**